MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -4356,15 +6447,172 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				INFOPLIST_FILE = "$(SRCROOT)/test/app/mac/Info.plist";
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/../Frameworks",
+				);
 				MACOSX_DEPLOYMENT_TARGET = 10.13;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.MSALMacTestApp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				SDKROOT = macosx;
-				USER_HEADER_SEARCH_PATHS = "$(inherited) $IDCORE_PATH/src/** $(HOME)/aadoverrides";
+				USER_HEADER_SEARCH_PATHS = (
+					"$(inherited)",
+					"$IDCORE_PATH/src/**",
+					"$(HOME)/aadoverrides",
+				);
+			};
+			name = Release;
+		};
+		28D1D57329BF62E900CE75F4 /* Debug */ = {
```

Comment:
This was added automatically by Xcode when the xcscheme for Integration Tests was added